templates/bin_setup

```diff
@@ -10,6 +10,9 @@ set -e
 gem install bundler --conservative
 bundle check || bundle install
 
+# Install JavaScript dependencies
+bin/yarn
```

Comment:
Will there always be a `bin/yarn`?

Reply:
Based on our conversation IRL, it seems like the answer is yes

Reply:
Yes, it gets run on `suspenders new` to install Normalize.css.